samples/client/petstore/rust/src/apis/add_pet_api.rs

```diff
@@ -0,0 +1,27 @@
+use hyper;
+use serde_json;
+use futures::{Future, Stream};
+use futures;
+
+use super::models;
+use super::Error;
+
+pub fn add_pet<C: hyper::client::Connect>(
+    prefix: &str,
```

Comment:
I'm not sure about what's the common pattern for api use, really. This sample client will require two things: a http (or https) prefix as a string and the hyper.rs Client (the web client) plus any arguments.